### ISSUE TITLE
🔤 Instructions using directional language

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#7375]
## ⌨ Accessibility Issue

Instructions provided for understanding and operating content rely solely on sensory characteristics of components, such as their visual location.

### WCAG 2 Criteria

1.3.3 Sensory Characteristics Level A
> Instructions for understanding and operating content do not rely solely on sensory characteristics of components such as shape, color, size, visual location, orientation, or sound.

REF: https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0#qr-content-structure-separation-understanding

## 🦋 Expected Behaviour

There are no references in the codebase that mention the position of an object.

## 🕵️ Details

User-facing copy using **below**:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Skills/SkillShowcasePage.tsx#L238

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/components/EmploymentEquity/dialogs/IndigenousDialog.tsx#L99

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Auth/CreateAccountPage/CreateAccountPage.tsx#L261

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/AccessibilityStatementPage/AccessibilityStatementPage.tsx#L622

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Applications/ApplicationSelfDeclarationPage/SelfDeclaration/CommunitySelection.tsx#L266

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Pools/EditPoolPage/components/OtherRequirementsSection/OtherRequirementsSection.tsx#L200

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Pools/BrowsePoolsPage/components/OngoingRecruitmentSection/OngoingRecruitmentSection.tsx#L1005

https://github.com/GCTC-NTGC/gc-digital-talent/blob/491ad59d218ceebcd40843c2a4c3a62f685259ca/apps/web/src/pages/Applications/ApplicationCareerTimelineAddPage/ApplicationCareerTimelineAddPage.tsx#L88

## 🙋‍♀️ Proposed Solution

Remove references of above, below, etc., in the codebase that mention the position of an object and replace them with words like preceding, following, etc.

## ✅ Acceptance Criteria

A set of assumptions verifying that the issue was addressed when tested.

- [ ] no references in the codebase that mention the visual position of an object

**Comment by petertgiles** [comment]
What is the scope of this issue?   The screenshots above show hits from inline code comments, auxiliary documentation, and UI from the application.

**Comment by mnigh** [comment]
> What is the scope of this issue? The screenshots above show hits from inline code comments, auxiliary documentation, and UI from the application.

I would say everything there is in the scope!

**Comment by RM-1978** [comment]
Agreed @mnigh

**Comment by gobyrne** [comment]
Thanks for digging up this list. Here's my take:
There's broad agreement that we should try to avoid directional language in our copy. However, I'm not sure it's helpful to do it to the extent of not allowing the word "below" in these context.

The reason that I feel it does not fail 1.3.3 is that the content is also below in the tab order and screen reader users (for example) are able to understand and navigate the page properly in response to reading "below". This was the opinion of an accessibility expert consultant I worked with back at NRCan. Also, I don't think the drive to change these is coming from users, if they are complaining it would change my approach here.

I tried to google best practice here. I didn't get what I was looking for, but instead I got a whole bunch of example from leading accessibility practitioner organisations where they use the word "below" in the same context as we do on their websites. Everyone is doing exactly the same thing as us!

https://www.google.com/search?q=accessibility+can+I+say+%22below%22&rlz=1C1GCEA_enCA884CA884&oq=accessibility+can+I+say+%22below%22&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRigATIHCAMQIRigAdIBCDg3NzVqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8&safe=active&ssui=on#ip=1

**My conclusion:**
I don't think we need to address these and in general using below should be avoided when convenient, but otherwise allowed. IF anyone feels so strongly about this that they want to change it anyway, I'm ok with that to. Please propose new copy for the list of occurrences that Matt pulled here. The new copy has to be equally or more clear than the existing copy though or I will not approve it. We can't afford to sacrifice clarity for the sake of trying to be purists.

**Comment by petertgiles** [comment]
Closing.  Please handle on a case-by-case basis with new copy as requested by @gobyrne
